ios/Podfile.lock

```diff
@@ -874,4 +874,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: c422aedd0c3b236578801088c8afef28030c3c83
 
-COCOAPODS: 1.13.0
+COCOAPODS: 1.12.0
```

Comment:
Why downgrade?

Reply:
Oh! My brain has bugged. I inverted the understanding here. I will double check.

Reply:
I removed this update.